autorest/azure/azure.go

```diff
@@ -147,6 +149,40 @@ func IsAzureError(e error) bool {
 	return ok
 }
 
+// Resource contains details about an Azure resource
```

Comment:
Can you add a link to the docs on Azure resource IDs (the one you had sent me).

Reply:
👍